src/App.tsx

```diff
@@ -1,61 +1,36 @@
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { NewTodo } from './components/NewTodo';
+
+import { Todo } from './Types/Todo';
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+
+  const addTodo = (todo: Todo) => {
+    setTodos(currentTodo => [...currentTodo, todo]);
+  };
+
+  const todosWithUser: Todo[] = todos.map(todo => {
+    const user = usersFromServer.find(person => person.id === todo.userId);
+
+    return { ...todo, user };
+  });
+
+  const arrayTodosId = todosWithUser.map(todo => todo.id);
+  const maxTodoId = Math.max(...arrayTodosId);
```

Comment:
If `todosWithUser` is empty, `Math.max(...arrayTodosId)` will return `-Infinity`, which could cause issues when generating new IDs. Consider providing a default value to `Math.max` to handle this case, such as `Math.max(0, ...arrayTodosId)`.